src/apps/beefy/shortcuts.ts

```diff
@@ -111,6 +111,56 @@ const hook: ShortcutsHook = {
             depositTx.estimatedGasUse = DEFAULT_DEPOSIT_GAS / 3n
           }
 
+          return { transactions }
+        },
+      }),
+      createShortcut({
+        id: 'withdraw',
+        name: 'Withdraw',
+        description: 'Withdraw your assets',
+        networkIds: [networkId],
+        category: 'withdraw',
+        triggerInputShape: {
+          tokens: tokenAmounts.length(1),
+          positionAddress: ZodAddressLowerCased,
+          tokenDecimals: z.coerce.number(),
```

Comment:
applies to the previous PR too, can you comment what args will be passed via `shortcutTriggerArgs`? also we'd need to update the beefy positions to return availableShortcutIds and shortcutTriggerArgs, not sure if that's captured in any issue

Reply:
should the updates to the position be done in [ACT-1369](https://linear.app/valora/issue/ACT-1369/update-beefy-hooks-positions-with-earn-data) where the other changes to positions.ts happen?

Reply:
yes, that sounds good, can you add details to the issue so we don't forget?